packages/volto/src/components/theme/RelatedItems/RelatedItems.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * RelatedItems component.
+ * @module components/theme/RelatedItems/RelatedItems
+ */
+
+import React from 'react';
+import { UniversalLink } from '@plone/volto/components';
+import { defineMessages, injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Container } from 'semantic-ui-react';
+
+const messages = defineMessages({
+  relatedItems: {
+    id: 'Related Items',
+    defaultMessage: 'Related Items',
+  },
+});
+
+/**
+ * Related Items component.
+ * @function RelatedItems
+ * @param {array} tags Array of related items.
+ * @returns {string} Markup of the component.
+ */
+
+const RelatedItems = ({ relatedItems, intl }) =>
+  relatedItems && relatedItems.length > 0 ? (
+    <Container>
+      <h4 className="ui header">{intl.formatMessage(messages.relatedItems)}</h4>
```

Comment:
I also changed the HTML to look more like `volto-relateditems-block`. Now the Related Items will look like this:

![download (5)](https://github.com/user-attachments/assets/2d497dad-f5a7-47a7-a894-f6d027c64262)

I didn't include the CSS because it didn't make a difference.

Reply:
> Done. I also changed the HTML to look more like `volto-relateditems-block`. Now the Related Items will look like this:
> 
> ![download (5)](https://private-user-images.githubusercontent.com/3297795/353993725-2d497dad-f5a7-47a7-a894-f6d027c64262.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI5NTIyMTcsIm5iZiI6MTcyMjk1MTkxNywicGF0aCI6Ii8zMjk3Nzk1LzM1Mzk5MzcyNS0yZDQ5N2RhZC1mNWE3LTQ3YTctYTg5NC1mNmQwMjdjNjQyNjIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MDZUMTM0NTE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjJmYTA2NTNhNWRjYTQwZjJmYmY1OTQyZjczNTg2MDA5MTkzNGY4MmFiMTYwYjE2MTNkYWM0MGQyOWQyZTM5NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.7cIk2kp2wktQgk7x3tN97bQyI3lUYmD6U_68yiPZpzQ)
> 
> I didn't include the CSS because it didn't make a difference.

@wesleybl if this is the final render update the screenshot from the ticket description with this or the latest render.